kong/plugins/oauth2/migrations/004_200_to_210.lua

```diff
@@ -0,0 +1,18 @@
+return {
+  postgres = {
+    up = [[
+      DO $$
+      BEGIN
+        ALTER TABLE IF EXISTS ONLY oauth2_credentials ADD hash_secret BOOLEAN DEFAULT FALSE;
```

Comment:
I believe adding the default on fields on the migrations like this is not the recommended way, because then it can conflict with the schema default in the future.

```suggestion
        ALTER TABLE IF EXISTS ONLY oauth2_credentials ADD hash_secret BOOLEAN;
```